docs/source/en/perf_train_gpu_many.md

```diff
@@ -465,66 +566,20 @@ We have 10 batches of 512 length. If we parallelize them by attribute dimension
-
-
-## Which Strategy To Use When
-
-Here is a very rough outline at which parallelism strategy to use when. The first on each list is typically faster.
-
-**⇨ Single GPU**
-
-* Model fits onto a single GPU:
-
-    1. Normal use
-
-* Model doesn't fit onto a single GPU:
-
-    1. ZeRO + Offload CPU and optionally NVMe
-    2. as above plus Memory Centric Tiling (see below for details) if the largest layer can't fit into a single GPU
-
-* Largest Layer not fitting into a single GPU:
-
-1. ZeRO - Enable [Memory Centric Tiling](https://deepspeed.readthedocs.io/en/latest/zero3.html#memory-centric-tiling) (MCT). It allows you to run arbitrarily large layers by automatically splitting them and executing them sequentially. MCT reduces the number of parameters that are live on a GPU, but it does not affect the activation memory. As this need is very rare as of this writing a manual override of `torch.nn.Linear` needs to be done by the user.
-
-**⇨ Single Node / Multi-GPU**
-
-* Model fits onto a single GPU:
-
-    1. DDP - Distributed DP
-    2. ZeRO - may or may not be faster depending on the situation and configuration used
-
-* Model doesn't fit onto a single GPU:
-
-    1. PP
-    2. ZeRO
-    3. TP
-
-    With very fast intra-node connectivity of NVLINK or NVSwitch all three should be mostly on par, without these PP will be faster than TP or ZeRO. The degree of TP may also make a difference. Best to experiment to find the winner on your particular setup.
-
-    TP is almost always used within a single node. That is TP size <= gpus per node.
-
-* Largest Layer not fitting into a single GPU:
-
-    1. If not using ZeRO - must use TP, as PP alone won't be able to fit.
-    2. With ZeRO see the same entry for "Single GPU" above
-
-
-**⇨ Multi-Node / Multi-GPU**
-
-* When you have fast inter-node connectivity:
-
-    1. ZeRO - as it requires close to no modifications to the model
-    2. PP+TP+DP - less communications, but requires massive changes to the model
+One very important aspect is that FlexFlow is designed for optimizing DNN parallelizations for models with static and 
+fixed workloads, since models with dynamic behavior may prefer different parallelization strategies across iterations.
 
-* when you have slow inter-node connectivity and still low on GPU memory:
+So the promise is very attractive - it runs a 30min simulation on the cluster of choice and it comes up with the best 
+strategy to utilise this specific environment. If you add/remove/replace any parts it'll run and re-optimize the plan 
+for that. And then you can train. A different setup will have its own custom optimization.
 
-    1. DP+PP+TP+ZeRO-1
+🤗 Transformers status: not yet integrated. We already have our models FX-trace-able via [transformers.utils.fx](https://github.com/huggingface/transformers/blob/master/src/transformers/utils/fx.py), which is 
```

Comment:
I think it's cool to mention FlexFlow, but if it isn't integrated in Transformers then I'm not sure we need to add this section here because it doesn't really help users who want to try it out but it's not available. 

Also not really a fan of "so someone needs to figure out what needs to be done to make FlexFlow work with our models". The tone sounds a bit demanding and not as friendly (totally understand this was already here before you started working on it 🙂 ).

Reply:
Rephrased the sentence. I'm on the fence about removing the content, or moving it to another section.